static/js/EN/EN.js

```diff
@@ -575,8 +575,9 @@ $('.inputbox').focusout(function() {
 
 $('#vel_a').focusout(function() {
     update();
-    updateBounds();
+    // updateBounds();
```

Comment:
Remove?

Reply:
this is was was causing issue #9. When user was selecting air velocity the graph was updating the thermal comfort bounds, but Stefano raise that as an issue.